docs/csv/template_files/ballot_measure_contest.txt

```diff
@@ -0,0 +1,23 @@
+abbreviation,An abbreviation for the Ballot Contest
+ballot_measure_selection_ids,References the relevant selection object
+ballot_sub_title,Subtitle of the Ballot item
+ballot_title,Title of the Ballot item
+electoral_district_id,References the relevant electoral district
+electorate_specification,Specifies any changes to the eligible electorate for this contest past the usual “all registered voters” electorate.
+external_identifier_type,Specifies the type of identifier. external_identifier_othertype. Allows for cataloging an ExternalIdentifier type that falls outside the options listed in IdentifierType. Type should be set to “other” when using this field.
+external_identifier_value,Specifies the identifier.
+has_rotation,Indicates whether the selections in the contest are rotated.
+name,Name of the contest not necessarily how it appears on the ballot
+sequence_order,Order in which the contests are listed on the ballot. 
+vote_variation,Vote variation associated with the contest
+other_vote_variation,
+con_statement,Specifies a statement in opposition to the referendum. 
+effect_of_abstain,Specifies what effect abstaining will have
+full_text,Specifies the full text of the referendum as it appears on the ballot.
+info_uri,Specifies a URI that links to additional information about the referendum.
+passage_threshold,Specifies the threshold of votes that the referendum needs in order to pass.
+pro_statement,Specifies a statement in favor of the referendum. 
+summary_text,Specifies a short summary of the referendum that is on the ballot below the title but above the text.
+type,Specifies the particular type of ballot measure. 
+other_type,Allows for cataloging a new
+id
```

Comment:
Not intentional. Just something I overlooked. I made no change to the ordering of the csv headers stored [here](https://github.com/votinginfoproject/csv-templates/tree/master/vip_5.1_csv_templates). I'm happy to update these so the ID placement is uniform.